### PR TITLE
completed dom-querying

### PR DESCRIPTION
**Pull request by Jc-Leon**
Screen Shot 2021-07-06 at 11 05 16 AM

### CODE REVIEW

File: dom-querying/main.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+
+document.querySelector("h1")
```
Comment (codingcodymiller) [inline dom-querying/main.js:2]: This line isn't necessary, you aren't saving the return anywhere.

File: dom-querying/main.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+console.log(document.querySelector("h1"))
+console.dir(document.querySelector("h1"));
+
+document.querySelector("#explanation")
```
Comment (codingcodymiller) [inline dom-querying/main.js:6]: This line isn't necessary, you aren't saving the return anywhere.

File: dom-querying/main.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+console.log(document.querySelector("#explanation"));
+console.dir(document.querySelector("#explanation"));
+
+document.querySelector(".hint")
```
Comment (codingcodymiller) [inline dom-querying/main.js:10]: This line isn't necessary, you aren't saving the return anywhere.

File: dom-querying/main.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+console.log(document.querySelector(".hint"));
+console.dir(document.querySelector(".hint"));
+
+document.querySelectorAll("p")
```
Comment (codingcodymiller) [inline dom-querying/main.js:14]: This line isn't necessary, you aren't saving the return anywhere.

File: dom-querying/main.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+console.log(document.querySelectorAll("p"))
+
+document.querySelectorAll(".example-link")
```
Comment (codingcodymiller) [inline dom-querying/main.js:17]: This line isn't necessary, you aren't saving the return anywhere.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: dom-querying/main.js (hunk 5 above)
Comment (Jc-Leon) [reply dom-querying/main.js]: whoops sorry about that i think overlooked at what the instructions were stating 😓 thank you for clarifying it more

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**